projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/service/deploy/DataJobDeploymentCrudITV2.java

```diff
@@ -200,6 +200,19 @@ public void testSynchronizeDataJobs() throws Exception {
     OffsetDateTime lastDeployedDateInitial = actualDataJobDeployment.getLastDeployedDate();
     Assertions.assertNotNull(deploymentVersionShaInitial);
     Assertions.assertNotNull(lastDeployedDateInitial);
+
+    jobsRepository.deleteById(testJobName);
```

Comment:
Nit: why not use JobService.delete() ? Or even the controller ? Isn' that more appropriate and realistic . 


Also we can test jobService.Delete ; jobService.create. (same-job-name) same-job-name should have zero deployments (eventually) . The test should pass because JobService.delete also deletes the desired deployment. But better tested for regressions than not.

Reply:
We are going to remove this test and use the controller as part of the DataJobDeploymentCrudAsyncIT.